xml/sle_update_preparation.xml

```diff
@@ -268,10 +268,26 @@
     <command>pg_upgrade</command> makes the migration less time-consuming.
    </para>
    <para>
-    Each &postgresql; version stores its files in different, version-dependent
-    directories. After the update the directories will change from
-    <filename>/usr/lib/postgresql96/</filename> to
-    <filename>/usr/lib/postgresql10/</filename>.
+    The program files for each PostgreSQL version are stored in different,
+    version-dependent directories. For example, in <filename>/usr/lib/postgresql96/</filename>
+    for version 9.6 and in <filename>/usr/lib/postgresql10/</filename> for version 10.
+    Note that the versioning policy of &postgresql; has changed between the major
+    versions 9.6 and 10. For details, see <link
+    xlink:href="https://www.postgresql.org/support/versioning/"/>.
+   </para>
+   <important>
+    <title>Upgrading from &slea; 11</title>
+     <para>
+      When upgrading from &slea; 11, <systemitem>postgresql94</systemitem> will
+      be uninstalled and cannot be used for the database migration to a higher
+      &postgresql; version. Therefore in this case make sure to migrate the
+      &postgresql; database <emphasis>before</emphasis> you upgrade your system.
+    </para>
+   </important>
+   <para>
+    The procedure below describes the database migration from version 9.6 to 10. When
+    using a different version as start or target, replace the version numbers
+    accordingly.
```

Comment:
Maybe add a hint that different instructions may apply if you are using different versions..?